test/LearningTestTool/py/kht_export.py

```diff
@@ -214,6 +215,70 @@ def export_learning_test_tree(
                                                     used_dataset_collections[
                                                         dataset_collection
                                                     ][dataset_name] = True
+                        # On traite de facon heuristique le cas d'un fichier de parametrage json
+                        # C'est en fait complexe a analyser dans le cas general en raison des search/replace
+                        # entre fichier de parametre json et fichier de scenario. Mais, c'est inutile d'avoir
+                        # une implementation rigouruse pour les usages basiques utilises dans les suites de test
+                        test_json_path = os.path.join(source_test_dir, kht.TEST_JSON)
+                        try:
+                            if os.path.isfile(test_prm_path):
+                                with open(
+                                    test_json_path, "r", errors="ignore"
```

Comment:
The JSON file is assumed as encoded in UTF-8, right? In this case, then there is nothing to do.

Reply:
> The JSON file is assumed as encoded in UTF-8, right? In this case, then there is nothing to do.

This indeed seems to be the case (see https://github.com/KhiopsML/khiops/wiki/Feature-V11:-Extension-du-pilotage-de-Khiops-via-les-sc%C3%A9narios#choix-dencodage). Hence, closing the thread.

Reply:
Un json correct doit être encodé en UTF-8 (la sous partie Base64 est ascii).
Par contre, la méthode est ici utilisée pour gérer de façon heuristique l'export, mais on peut avoir à gérer des json incorrects.

Rajout du commentaire suivant:
```
    # On ignore ici les erreurs dans cette methode d'export "pragmatique", pour ne pas etre
    # bloque par des json errones (codage non utft-8) utilises pour tester la gestion
    # des erreurs dans KhiopsKhiops
```

Reply:
Okay for me.